### PR TITLE
Simplify k8s instructions

### PR DESCRIPTION
**Pull request by lukemarsden**
Removing workarounds no longer needed in modern clusters, streamline/simplify the instructions, and recommend Weave Cloud usage.

PTAL @paulbellamy. Thanks!

### CODE REVIEW

File: site/installing.md [hunk 1]
```diff
@@ -177,27 +177,21 @@ After Scope has been launched, open your web browser to [https://cloud.weave.wor
 
 ##<a name="k8s"></a>Installing Scope on Kubernetes
 
-To install Scope onto a Kubernetes cluster in Standalone Mode, follow these instructions.
+### With Weave Cloud (recommended)
 
-**Before You Begin**
+Weave Cloud runs the Scope UI for you, so you don't have to configure exposing and securing it for your team.
```
Comment (rade) [inline site/installing.md]: plus keeping it running. Though the whole "so you don't have to configure exposing and securing it" reads awkwardly already.

Comment (paulbellamy) [inline site/installing.md]: maybe, "Weave Cloud runs the Scope UI for you, and provides secure access control for your team."

File: site/installing.md [hunk 2]
```diff
@@ -177,27 +177,21 @@ After Scope has been launched, open your web browser to [https://cloud.weave.wor
...
-
-**Install Scope on Your Cluster**
-
-You can run Scope natively in your Kubernetes cluster using `kubectl` with a launch URL as show below.
+### Without Weave Cloud (run the Scope UI yourself)
 
 The simplest way to get the latest release of Scope deployed onto a Kubernetes cluster is by running the following:
 
-    kubectl create -f 'https://cloud.weave.works/launch/k8s/weavescope.yaml' --validate=false
+    $ kubectl apply -f 'https://cloud.weave.works/launch/k8s/weavescope.yaml'
```
Comment (errordeveloper) [inline site/installing.md]: This will close https://github.com/weaveworks/scope/issues/1854 👍

**Comment by errordeveloper** [comment]
@paulbellamy I've updated this with the wording you have suggested, PTAL.

**Pull request merged**